File: submissions/sm_112_soumik/week_03/day_4/session_1/check_superset.js
function checkSub(input1,input2){
    var sub=[]
    var count = 0
    for(i =0; i< input1.length;i++){
        for(j =0;j<input2.length;j++)
        if(input1.includes (input2[j]))
            count += 1       
    }
    if(count == 0){
        console.log("Not Superset")
    }
    else
        console.log(" Superset")
}
checkSub([10,12,15,18,0,5],[19,0,5])

File: submissions/sm_105_ashish/week_13/day_4/session_2/check_superset.py
set1 = {1,2,3,4,5,6}
set2 = {1,2,6}
count = 0
for i in set1:
    for j in set2:
        if i == j:
            count += 1
if count == len(set2):
    print("set1 is a superset of set2")
else:
    print("set1 is not a superset of set2")